tests/core/test_stuff.py

```diff
@@ -388,3 +388,51 @@ def test_external_trigger_experiment_state():
     assert sprev.unique().count() == 48750
     assert sthis.unique().count() == 92090
     assert snext.unique().count() == 48751
+
+
+def test_steps():
```

Comment:
maybe test_trim_dead_ends

Reply:
I have mixed feelings about this test. On the one hand, it would be a lot faster if you just construct some dummy steps with whatever sequence of outputs you want and try trimming them. On the other hand, this tests that it works for things that are relevant for us?  I think I lean towards just testing on dummy steps without making a whole channel and doing learning steps.